test/jest/__mock__/stripesSmartComponents.mock.js

```diff
@@ -4,6 +4,31 @@ jest.mock('@folio/stripes/smart-components', () => ({
   ClipCopy: jest.fn(() => null),
   makeQueryFunction: jest.fn((value) => value),
   CheckboxFilter: jest.fn(() => null),
+  NoteCreatePage: (props) => {
+    return (
+      <div>
+        <div>NoteCreatePage</div>
+        {props.renderReferredRecord()}
+      </div>
+    );
+  },
+  NoteEditPage: (props) => {
+    return (
+      <div>
+        <div>NoteEditPage</div>
+        {props.renderReferredRecord()}
+      </div>
+    );
+  },
```

Comment:
I do not see that NoteCreatePage and NoteEditPage are used inside `NoteViewRoute` component.
Could you remove them?

Reply:
Changed done